tests/func/test_data_cloud.py

```diff
@@ -149,6 +152,21 @@ def get_ssh_url():
     )
 
 
+def get_ssh_url_mocked(user, port):
+    if os.name == "nt":
+        # since we use posixpath to compose paths.
```

Comment:
I believe this comment could be improved :slightly_smiling_face:

Reply:
What do you want to improve? I'm not a native speaker, so description sometimes not ideal for me.